### DISCUSSION TITLE
Function inputs and escaping special characters with Unicode

### DISCUSSION THREAD

**Discussion by glorious-beard** [microsoft/semantic-kernel#7308] [Q&A] [answered]
If I set a kernel argument to content containing special characters, (HTML tags, for example), and I look at the logger output from the kernel when it's invoking the function, I notice that the JSON object escapes all of the special character.

For example, if I set "input" to "<p>Version 1.2</p>....", the function argument looks like:
```json
{"input":"\u003Cp\u003EVersion 1.2\u003C/p\u003E..."}
```

Two questions:
1. Do the extra characters in escaping "<" and ">" with 5 additional characters incur extra token cost?
2. Does the function call unescape these characters before it is sent to the LLM endpoint?

**Comment by eavanvalkenburg** [comment, marked as answer]
Answered here: https://github.com/microsoft/semantic-kernel/issues/7365
